pom.xml

```
@@ -176,11 +206,18 @@
                     <excludes>
                         <exclude>**/uz/enterprise/mytex/MytexApplication.class</exclude>
                         <exclude>**/uz/enterprise/mytex/audit/**/*</exclude>
-                        <exclude>**/uz/enterprise/mytex/config/**/*</exclude>
+                        <exclude>**/uz/enterprise/mytex/configs/**/*</exclude>
                         <exclude>**/uz/enterprise/mytex/constant/**/*</exclude>
                         <exclude>**/uz/enterprise/mytex/entity/**/*</exclude>
                         <exclude>**/uz/enterprise/mytex/repository/**/*</exclude>
-                        <exclude>**/uz/enterprise/mytex/exception/**/*</exclude>
+                        <exclude>**/uz/enterprise/mytex/exceptions/**/*</exclude>
+                        <exclude>**/uz/enterprise/mytex/dto/**/*</exclude>
+                        <exclude>**/uz/enterprise/mytex/enums/**/*</exclude>
+                        <exclude>**/uz/enterprise/mytex/filters/**/*</exclude>
```

```suggestion
                        <exclude>**/uz/enterprise/mytex/filter/**/*</exclude>
```
use singular nouns for packages